src/bundle/Controller/SearchController.php

```diff
@@ -209,6 +216,7 @@ public function searchAction(Request $request): Response
             if ($result instanceof Response) {
                 return $result;
             }
+            $this->entrypointLookupCollection->getEntrypointLookup('ezplatform')->reset();
```

Comment:
I wonder if we should reset also the `default` build?

Reply:
I tested this with `ezplatform` only and it works fine.